### DISCUSSION TITLE
Server.key

### DISCUSSION THREAD

**Discussion by Akshit-ls** [JFC-Group/JF-Customisation#52] [Q&A] [answered]
Server.key in my router JCOW404, firmware 2.3 is I think in hex form, I have attached the file. What is the server.key in your router's firmware ?
[server.zip](https://github.com/user-attachments/files/16140088/server.zip)
The hexdump of file is 
0000000 e9be 755b 3e34 dc75 f25e 75b1 ca75 0a75

**Comment by itsyourap** [comment, marked as answer]
It's obviously different. I have a JCOW414.
The attached file [Keys.zip](https://github.com/user-attachments/files/16144701/Keys.zip) contains all the keys I had dumped previously from the firmware image.
